resources/widgetlist.ts

```diff
@@ -285,8 +320,8 @@ class WidgetList extends HTMLElement {
     if (!this._connected)
       return;
 
-    this.rootElement.style.width = this._rowcolsize * this._elementwidth;
-    this.rootElement.style.height = this._rowcolsize * this._elementheight;
+    this.rootElement.style.width = String(this._rowcolsize * this._elementwidth);
+    this.rootElement.style.height = String(this._rowcolsize * this._elementheight);
```

Comment:
I think not to use `String` constructor, directly invpke `toString()` would be better? also, a template string with number literal is not bad though. here and everywhere.